### ISSUE TITLE
QR Scanner Dont Works

### ISSUE THREAD

**Issue by gustavo-pin** [spesmilo/electrum#8883]
Device: Samsung Galaxy M52 5G
Download: PlayStore
Android Version: 13

**Comment by SomberNight** [comment]
Please provide more details.
In what way does it not work? What are you doing exactly, and what happens?
Did older versions of Electrum also exhibit the problem?

**Comment by gustavo-pin** [comment]
> Please provide more details.
> In what way does it not work? What are you doing exactly, and what happens?
> Did older versions of Electrum also exhibit the problem?

When I use the scan to make a payment or to import a public key it doesn't work.

**Comment by SomberNight** [comment]
I've just tested scanning the QR code from your video with the built-in scanner using my phone and it scanned it correctly. Perhaps the issue is specific to some devices or depends on the quality of the camera.

**Comment by accumulator** [comment]
I saw a noticable delay (eventually it took) when holding the scanner exactly level. When tilting a bit the detection was much faster.

**Comment by gustavo-pin** [comment]
> I've just tested scanning the QR code from your video with the built-in scanner using my phone and it scanned it correctly. Perhaps the issue is specific to some devices or depends on the quality of the camera.

For scanning in the Wallet of Satoshi app it works quickly, for pix and other use cases too. Think the problem is not on my hardware.

**Comment by accumulator** [comment]
I ran into this issue again scanning a QR code for a bip21 address-only code `bitcoin:3...` from a coinbase wallet.

**Comment by gustavo-pin** [comment]
> I ran into this issue again scanning a QR code for a bip21 address-only code `bitcoin:3...` from a coinbase wallet.

Did it work?

**Comment by accumulator** [comment]
Yes it was pretty bright. I managed to scan it successfully using https://github.com/SecUSo/privacy-friendly-qr-scanner which is also ZXing based.

**Comment by accumulator** [comment]
here another example of a difficult to scan QR
https://geyser.fund/project/nobsbitcoin/funding/payment/lightning
